lib/mayaUsd/fileio/shading/shadingModeUseRegistry.cpp

```diff
@@ -298,6 +298,10 @@ class UseRegistryShadingModeExporter : public UsdMayaShadingModeExporter
             }
         }
 
+        for (auto&& writerEntry : shaderWriterMap) {
+            writerEntry.second->PostExport();
+        }
+
```

Comment:
Allow exporters to finalize connections post-export. Used to link fileTexture and place2dTexture.